### PR TITLE
[b/352468287] Add non-CDB sql-stats-awr

### PR DESCRIPTION
**Pull request by misolt**
- enable non-CDB queries for AWR
- add non-CDB sql-stats-awr

### CODE REVIEW

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/StatsTaskListGenerator.java [hunk 1]
```diff
@@ -89,30 +102,30 @@ ImmutableList<Task<?>> createTasks(ConnectorArguments arguments, Duration querie
       builder.add(StatsJdbcTask.fromQuery(query));
     }
     for (String name : NATIVE_NAMES_OPTIONAL) {
-      builder.addAll(createTaskWithAlternative(name, /* isRequired= */ false, queriedDuration));
+      QueryGroup group = QueryGroup.create(/* isRequired= */ false, NATIVE, SINGLE_TENANT);
```
Comment (dawidxc) [inline dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/StatsTaskListGenerator.java:105]: I'm looking at the `QueryGroup` class and the first parameter to `create` method is `required`, not `isRequired`.

**Review by dawidxc — APPROVED** [review]
Approval for style only.

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/StatsTaskListGenerator.java (hunk 1 above)
Comment (misolt) [reply dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/StatsTaskListGenerator.java]: Ah, true. The literals' comments keep popping up, plain VS Code isn't much help with them. I fixed it to say `required` consistently

File: dumper/app/src/main/resources/oracle-stats/dba/awr/sql-stats-awr.sql [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    sum(A.end_of_fetch_count_delta) "EndOfFetchCountTotal",
+    sum(A.executions_delta) "ExecutionsTotal",
+    sum(A.iowait_delta) "IoWaitTotal",
+    sum(A.javexec_time_delta) "JavaExecTotal",
+    sum(A.physical_read_bytes_delta) "PhysicalReadBytesTotal",
+    sum(A.physical_write_bytes_delta) "PhysicalWriteBytesTotal",
+    sum(A.plsexec_time_delta) "PlsExecTotal",
+    sum(A.px_servers_execs_delta) "PxExecutionsTotal",
+    sum(A.rows_processed_delta) "RowsTotal"
+FROM dba_hist_sqlstat A
```
Comment (cyulysses-corp) [inline dumper/app/src/main/resources/oracle-stats/dba/awr/sql-stats-awr.sql:40]: super nit: prefer having more representative shorthand names when possible. In this case, sqlstat and snapshot would be good.

File: dumper/app/src/main/resources/oracle-stats/dba/awr/sql-stats-awr.sql [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+JOIN dba_hist_snapshot B
+ON A.dbid = B.dbid
+    AND A.instance_number = B.instance_number
+    AND A.snap_id = B.snap_id
+    AND B.end_interval_time > sysdate - ?
```
Comment (cyulysses-corp) [inline dumper/app/src/main/resources/oracle-stats/dba/awr/sql-stats-awr.sql:45]: Need a comment to tell what this "?" will be replaced with.

**Review by cyulysses-corp — APPROVED** [review]

**Review by kpiotrowski — APPROVED** [review]

File: dumper/app/src/main/resources/oracle-stats/dba/awr/sql-stats-awr.sql (hunk 3 above)
Comment (misolt) [reply dumper/app/src/main/resources/oracle-stats/dba/awr/sql-stats-awr.sql:45]: Done, I added the comment in awr sqls.

File: dumper/app/src/main/resources/oracle-stats/dba/awr/sql-stats-awr.sql (hunk 2 above)
Comment (misolt) [reply dumper/app/src/main/resources/oracle-stats/dba/awr/sql-stats-awr.sql]: Done, I used the names `SqlStat` and `Snapshot`.

**Pull request merged**